### ISSUE TITLE
Don't add boilerplate for transformed docs

### ISSUE THREAD

**Issue by sebastianbenz** [ampproject/amppackager#335]
AMPPackager adds the boilerplate code for transformed AMP pages without (`<html amp i-amphtml-no-boilerplate ...`).

To reproduce: 

```
curl -si -H 'amp-cache-transform: google' -H 'accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3' https://amp.dev | grep -a boilerplate
```

includes the boilerplate: 

```
</script><link href=https://amp.dev/manifest.json rel=manifest><style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript></head><body class=ap--websites>
```

whereas:

```
curl https://amp.dev | grep -a boilerplate
```

doesn't.

For internal reference b/138961576

**Comment by sebastianbenz** [comment]
//cc @twifkak @Gregable

**Comment by honeybadgerdontcare** [comment]
Fixed in #334
